stacks/api/storage/storage.service.ts

```diff
@@ -80,13 +80,11 @@ export class Storage {
   insertMany(object: StorageObject[]): Promise<StorageObject[]> {
     const data = Array.from(object);
 
-    Promise.all(
-      data.map(d => {
-        d._id = new ObjectId(d._id);
-        fs.writeFileSync(this.buildPath(d), d.content.data);
-        delete d.content.data;
-      })
-    );
+    data.forEach(d => {
+      d._id = new ObjectId(d._id);
+      fs.writeFileSync(this.buildPath(d), d.content.data);
```

Comment:
For performance reasons can we replace this with fs.promises.writeFile instead? 
Cause we don't want to stop the event queue during IO operations.